### DISCUSSION TITLE
Is it possible to embed files from outside of /docs?

### DISCUSSION THREAD

**Discussion by evertonspader-tomtom** [docsifyjs/docsify#2408] [Q&A] [answered]
Hey all, in my repository I have my `/docs` directory where I keep all the documentation and at the same level I have a `/examples` directory with some files that are accessible for others as tutorials. I would like to embed these files in the documentation so that any changes made to the files are automatically reflected in the documentation but it seems that I can't embed files from outside `/docs`, is that correct? This is my directory structure:

```shell
.
├── docs
│   ├── README.adoc
└── examples
    ├── example1
    │   └── app1.py
    └── example2
        └── app2.py
```

It's worth noting that I am using `.adoc` files instead of `.md` and this is the error I'm getting:
```shell
Unresolved directive in <stdin> - include::../examples/example1/app1.py[]
```

**Comment by sy-records** [comment, marked as answer]
You need to put it in the docs directory.

If you don't want to put it in it, you need to provide an accessible url.

```
[filename](examples/example1/app1.py ':include :type=code')
```

```
.
└── docs
    ├── README.adoc
    └── examples
        ├── example1
        │   └── app2.py
        └── example2
            └── app2.py
```

**Reply by evertonspader-tomtom** [reply]
@sy-records I'm sorry for the confusion but you said I should move it to the docs directory but in your example it sits outside still. My question here is exactly that, if there is a way to embed files from outside because we don't want to mix "docs" with the "examples" directories.

**Reply by sy-records** [reply]
Oh, I'm sorry, I modified it wrong. It should be in the docs directory.

If you don't want to put it in it, you need to provide an accessible url.
